docs/pages/versions/unversioned/sdk/filesystem-next.mdx

```diff
@@ -10,14 +10,106 @@ platforms: ['android', 'ios', 'tvos']
 import APISection from '~/components/plugins/APISection';
 import { APIInstallSection } from '~/components/plugins/InstallSection';
 
-`expo-file-system` provides access to a file system stored locally on the device.
+> **INFO**
+> The `next` version of the filesystem API is included in the `expo-file-system` package. It can be used alongside the previous API, and offers a simplified, object oriented way of performing filesystem operations.
```

Comment:
```suggestion
> **info** The `next` version of the filesystem API is included in the `expo-file-system` library. It can be used alongside the previous API, and offers a simplified, object oriented way of performing filesystem operations.
```

Consistent wording and formatting.